packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -192,6 +216,86 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     }
   }
 
+  const memberships = await prisma.membership.findMany({
+    where: {
+      userId: ctx.user.id,
+    },
+  });
+
+  const teamIds = memberships.map((membership) => membership.teamId);
+
+  // Send webhook to notify other services
+  const subscriberOptions: GetSubscriberOptions = {
+    userId: ctx.user.id,
+    teamIds: teamIds,
+    orgId: ctx.user.organizationId,
+    triggerEvent: WebhookTriggerEvents.OOO_CREATED,
+  };
+
+  const subscribers = await getWebhooks(subscriberOptions);
+
+  await Promise.all(
+    subscribers.map(async (subscriber) => {
+      sendPayload(
+        subscriber.secret,
+        WebhookTriggerEvents.OOO_CREATED,
+        dayjs().toISOString(),
+        {
+          appId: subscriber.appId,
+          subscriberUrl: subscriber.subscriberUrl,
+          payloadTemplate: subscriber.payloadTemplate,
+        },
+        {
+          oooEntry: {
```

Comment:
To make sure that we use the same fields here as we return in `listOOOEntries`, can we create a shared select constant with all the fields? Then use this to fetch the newly created `outOfOfficeEntry` here and also use it in `listOOOEntries`